Samples/MyTestMacOSApp/MyTestMacOSApp/ADTestAppCache.m

```diff
@@ -141,7 +145,7 @@ - (OSStatus)readFromKeychain
         CFDataRef data = NULL;
         OSStatus status = SecItemCopyMatching((CFDictionaryRef)readQuery, (CFTypeRef *)&data);
         
-        _data = (__bridge NSData*)data;
+        _data = (__bridge_transfer NSData*)data;
```

Comment:
Just to have it in written too:
SecItemCopyMatching doesn’t need to own the incoming query dictionary, therefore bridge_retain is not necessary, since we’re using it in local scope. It will leak if you use bridge_retain on the query dictionary and don’t release it after using.